ui/src/DevUtils/MockVertx/MockVertxServer.js

```diff
@@ -212,7 +232,7 @@ const defaultConfig = {
   // consume records endpoint
   consumeEndpoint: CONSUME_ENDPOINT,
   // enable/disable extra logging
-  enableLog: false,
+  enableLog: true,
```

Comment:
Being hyper picky, should this really be enabled by default? I know we enabled it to debug the webkit websocket issue - was it useful to leave on?

Reply:
I missed that change - have reverted it.